cocos/base/CCScheduler.cpp

```diff
@@ -464,26 +464,27 @@ void Scheduler::schedulePerFrame(const ccSchedulerFunc& callback, void *target,
     HASH_FIND_PTR(_hashForUpdates, &target, hashElement);
     if (hashElement)
     {
+        CCAssert(!hashElement->entry->markedForDeletion, "All elements in _hashForUpdate must not be marked for deletion.");
```

Comment:
Then I think deleting this line is better, because that situation will never occur.